### ISSUE TITLE
in-document pragmas for classes without member definitions are ignored

### ISSUE THREAD

**Issue by jnsnow** [PyCQA/pylint#3099]
### Steps to reproduce

Consider the example from issue #3098, but extended to make use of the mixin:

```python
import abc
from collections import UserDict
from collections.abc import Mapping
import logging

class SomeDictMixin(abc.ABC, Mapping):
    def logged_get(self, key, default=None):
        tmp = self.get(key, default)
        logging.debug("Got %s for key %s", tmp, key)
        return tmp

class MyDict(SomeDictMixin, UserDict):
    """There are no methods to define!"""
```

This will produce a pylint error about the depth of ancestry:
`example.py:12:0: R0901: Too many ancestors (10/7) (too-many-ancestors)`

However, it cannot be silenced at the class level:

```python
class MyDict(SomeDictMixin, UserDict):
    # pylint: disable=too-many-ancestors
    """There are no methods to define!"""
    # pylint: disable=too-many-ancestors
```

### Current behavior

Pylint seemingly does not read pragmas when there is effectively only a `__doc__` body, so it yields the same error:

```
example.py:12:0: R0901: Too many ancestors (10/7) (too-many-ancestors)
```

There is a workaround, and hopefully it makes you chuckle:

```python
class MyDict(SomeDictMixin, UserDict):
    # pylint: disable=too-many-ancestors, unnecessary-pass
    """There are no methods to define!"""
    pass
```

### Expected behavior

Even though the class is hollow, it provides a useful composition to which we may still want to apply pragmas. In this case, the composition is between the standard library and a single user-defined class; so the actual logistical depth is closer to one or two. Users may wish to rightfully ignore pylint's suggestion.

### pylint --version output

```
pylint 2.3.1
astroid 2.2.5
Python 3.7.4 (default, Jul  9 2019, 16:32:37) 
[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)]
```

**Comment by PCManticore** [comment]
Thanks for creating the issue, I can reproduce it as well.

**Comment by Pierre-Sassoulas** [comment]
There was some fix to too-many-ancestors to count only the user defined ancestors, but it's still possible to reproduce using ``pylint a.py --enable=all --enable-all-extensions --max-parent=1``

The warning should be disabled this way:
```python
import abc
from collections import UserDict
from collections.abc import Mapping
import logging


class SomeDictMixin(abc.ABC, Mapping):
    def logged_get(self, key, default=None):
        tmp = self.get(key, default)
        logging.debug("Got %s for key %s", tmp, key)
        return tmp


class MyDict(SomeDictMixin, UserDict): # pylint: disable=too-many-ancestors
    """There are no methods to define!"""
```
